src/igvfd/tests/test_upgrade_publication.py

```diff
@@ -13,3 +13,9 @@ def test_publication_upgrade_1_2(upgrader, publication_v2):
     assert 'identifiers' not in value
     assert 'publication_identifiers' in value and value['publication_identifiers'] == ids
     assert value['schema_version'] == '3'
+
+
+def test_publication_upgrade_1_2(upgrader, publication_v3):
```

Comment:
test_publication_upgrade_1_2 should be test_publication_upgrade_3_4.
Unrelated but I think the above function should be renamed similarly as well.